docs/content/users/extend/customizing-images.md

```diff
@@ -178,6 +178,18 @@ An example of using `$TARGETARCH` would be:
 RUN curl --fail -JL -s -o /usr/local/bin/mkcert "https://dl.filippo.io/mkcert/latest?for=linux/${TARGETARCH}"
 ```
 
+## Adding unsupported PHP versions
```

Comment:
```suggestion
## Adding older PHP versions
```

"Unsupported" describes it from a DDEV point of view. 
I feel developer using DDEV would naturally search for "older PHP versions". Thoughts?

Reply:
I added more keywords: EOL, older, unsupported, unmaintained.